tools/SetupFlow/DevHome.SetupFlow/Views/SummaryView.xaml

```diff
@@ -212,7 +212,7 @@
                             BorderThickness="{ThemeResource TopLeftQuadrantThickness}">
                             <Grid>
                                 <Grid.RowDefinitions>
-                                    <RowDefinition Height="50"/>
+                                    <RowDefinition Height="70"/>

```

Comment:
why the additional 20?

Reply:
This is how it looked with 50 (the icon almost gets into previous line space, Didn't look nice to me):
![image](https://github.com/microsoft/devhome/assets/51001703/027c3e97-a192-49aa-834c-88e3e0cd526f)

This is with 70:
![image](https://github.com/microsoft/devhome/assets/51001703/b620881f-586d-4e6c-b655-9e9770eb3485)